.github/workflows/continuous-integration.yml

```diff
@@ -435,98 +539,76 @@ jobs:
-        continue-on-error: true
+      - name: Create report and post results to BigQuery
+        run: yarn cypress-mochawesome-to-bigquery
+        working-directory: testing-tools-team-dashboard-data
 
-      - name: Download test result artifact 2
-        uses: actions/download-artifact@v2
+      - name: Upload Mochawesome report artifact
+        uses: actions/upload-artifact@v2
         with:
-          name: cypress-mochawesome-test-results-artifact-2
-          path: testing-tools-team-dashboard-data/src/mochawesome/data
-        continue-on-error: true
+          name: mochawesome-report
+          path: testing-tools-team-dashboard-data/mochawesome-report
 
-      - name: Download test result artifact 3
-        uses: actions/download-artifact@v2
+      # --------------------------
+      # | Publish Testing Report |
+      # --------------------------
+
+      - name: Checkout Testing Reports repo
+        uses: actions/checkout@v2
         with:
-          name: cypress-mochawesome-test-results-artifact-3
-          path: testing-tools-team-dashboard-data/src/mochawesome/data
-        continue-on-error: true
+          repository: department-of-veterans-affairs/testing-reports
+          token: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
+          path: testing-reports
 
-      - name: Download test result artifact 4
+      - name: Download Mochawesome report artifact
         uses: actions/download-artifact@v2
         with:
-          name: cypress-mochawesome-test-results-artifact-4
-          path: testing-tools-team-dashboard-data/src/mochawesome/data
-        continue-on-error: true
+          name: mochawesome-report
+          path: testing-reports/cypress-reports
 
-      - name: Download test result artifact 5
+      - name: Download video artifacts
+        if: ${{ needs.cypress-tests.result == 'failure' }}
         uses: actions/download-artifact@v2
         with:
-          name: cypress-mochawesome-test-results-artifact-5
-          path: testing-tools-team-dashboard-data/src/mochawesome/data
-        continue-on-error: true
-
-      - name: Create report and post results to BigQuery
-        run: yarn cypress-mochawesome-to-bigquery
+          name: cypress-video-artifacts
+          path: testing-reports/cypress-reports/videos/${{ env.UUID }}
 
-  nightwatch-tests:
-    name: Nightwatch E2E Tests
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Checkout vets-website
-        uses: actions/checkout@v2
+      - name: Add changes
+        run: git add .
+        working-directory: testing-reports
 
-      - name: Download production build artifact
-        uses: actions/download-artifact@v2
+      - name: Commit and push changes
+        uses: actions-js/push@master
         with:
-          name: vagovprod.tar.bz2
+          github_token: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
+          repository: department-of-veterans-affairs/testing-reports
+          directory: testing-reports
```

Comment:
I've noticed that this had trouble uploading a larger test video previously (~360 MB), so the [workflow failed on this step](https://github.com/department-of-veterans-affairs/vets-website/pull/17791/checks?check_run_id=3013565117). Might be something to look into.

I also noticed in the artifact that there's a compressed video, but it may not be getting generated properly as I couldn't play it after downloading.